8Knot/cache_manager/cache_facade.py

```diff
@@ -152,16 +152,16 @@ def get_uncached(func_name: str, repolist: list[int]) -> list[int]:  # or None
             return not_cached
 
 
-def caching_wrapper(func_name: str, query: str, repolist: list[int]) -> None:
+def caching_wrapper(func_name: str, query: str, repolist: list[int], n_repolist_uses=1) -> None:
```

Comment:
whats this change?

Reply:
In the sql query for repo_files, the repolist is used twice, so we have to pass it as a 'var' to psycopg2 twice. I'm going to generalize this in the near future, but this works for the queries we currently have.

Reply:
Makes sense, just didnt want to approve something that I didnt understand. Add comment in to explain temp step

Reply:
It's already below in the Args list

Reply:
ah i see now, I misread the initial explanation at first